share/completions/gradle.fish

```diff
@@ -7,12 +7,16 @@ end
 
 function __fish_gradle_create_completion_cache_file
     # Set up cache directory
-    if test -z $XDG_CACHE_HOME
+    if command test -z $XDG_CACHE_HOME
```

Comment:
No thank you. We ship a test *builtin*, and calling that is both faster and not reliant on the operating system `test`. It's also, you know, shorter.

Reply:
This *should* be quoted tho - `if test -z "$XDG_CACHE_HOME"`

Reply:
Removed command infront of `test` and quoted `$XDG_CACHE_HOME`